test/Spec.hs

```diff
@@ -123,3 +124,59 @@ spec = do
     it "hour" $ flip shouldBe "00:01:00:00" $ format diffComponents 3600
     it "day" $ flip shouldBe "01:00:00:00" $ format diffComponents 86400
 
+  describe "list formatters" $ do
+    it "concatenated" $ flip shouldBe "onetwothree" $ format (concatenated text) ["one", "two", "three"]
```

Comment:
Does the precedence of the operators allow this to be 
```suggestion
    it "concatenated" $ format (concatenated text) ["one", "two", "three"] `shouldBe` "onetwothree"
```
or would it need 
```suggestion
    it "concatenated" (format (concatenated text) ["one", "two", "three"] `shouldBe` "onetwothree")
```
(which isn't too gross IMO)